frontend/src/features/books/EditBookForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { API_URL } from '../../constants'
+
+function EditBookForm() {
+    const [book, setBook] = useState(null)
+    const [title, setTitle] = useState('')
+    const [author, setAuthor] = useState('')
+    const [description, setDescription] = useState('')
```

Comment:
@nshelokar-vf, why we are declaring & using multiple states here?
Eventually, we are using all the attributes from a book only. Please address accordingly

Reply:
Okay I will do that.